### DISCUSSION TITLE
My neorg has become absolutely slow. Need help.

### DISCUSSION THREAD

**Discussion by txtyash** [nvim-neorg/neorg#271] [General]
Opening times for norg files are about 3secs and adding new lines, navigating,
etc. are extremely slow.
I thought something must be wrong with my config for neorg but with minimal
config too my neorg is extremely slow.

**Comment by mrossinek** [comment]
Concealing is not the most performant right now but we re-enabled it yesterday. If you absolutely want to disable this, use this in your config:
```lua
        ["core.norg.concealer"] = {
            config = {
                markup = {
                    enabled = false,
                },
            },
        },
```

A PR to improve the performance is in the works: https://github.com/nvim-neorg/neorg/pull/267

**Reply by txtyash** [reply]
I did it but the problem persists.

**Reply by vhyrro** [reply]
Hey @zim0369, we've just merged our performance improvements. Although it's not bug-free it's definitely much faster. The performance improvements also affect Neorg as a whole, not just the concealer. Hope you find it nicer now :)
